formulas/dnsmasq/README.md

```diff
@@ -13,7 +15,7 @@ Examples
 
     wickFormula dnsmasq --start
 
-Returns nothing
+    wickFormula dnsmasq --protect-dns
```

Comment:
This worries me because we should have a section that says

    Returns nothing.

because the Returns section is not optional in TomDoc. Would you add that to the README.md file and regenerate documentation?